src/expression/functions.ts

```diff
@@ -743,6 +754,11 @@ export namespace DefaultFunctions {
         .add2("null", "*", () => null)
         .build();
 
+    export const unique = new FunctionBuilder("unique")
+        .add1("array", arr => arr.filter((v, i, s) => s.indexOf(v) === i))
```

Comment:
This implementation is quadratic time and will only work for numberic/string primitive types - DataArray#unique has an alternative implementation which uses a sort followed by a filter, which I recommend using.